ui/src/features/annotator/tools/segment-anything-tool/interactive-segmentation-point.component.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+import { InteractiveAnnotationPoint } from './segment-anything.interface';
+
+interface InteractiveSegmentationPointProps extends InteractiveAnnotationPoint {
+    isLoading: boolean;
+}
+
+export const InteractiveSegmentationPoint = ({ x, y, positive, isLoading }: InteractiveSegmentationPointProps) => {
+    const fill = positive ? 'var(--brand-moss)' : 'var(--brand-coral-cobalt)';
+    const radius = `calc(1 / var(--zoom-scale))`;
```

Comment:
Line 19 also has a radius calculation, but with a different value. Could you use more descriptive variable names so it’s clear what each one is used for?

Reply:
Updated